manifests/istio-control/istio-autoinject/files/injection-template.yaml

```
@@ -319,12 +319,16 @@ template: |
         add:
         - NET_ADMIN
         {{- end }}
+        {{ if eq (annotation .ObjectMeta `sidecar.istio.io/interceptionMode` .ProxyConfig.InterceptionMode) `NONE` -}}
+        add:
+        - CAP_NET_BIND_SERVICE
```

I think this is fine from security perspective CC @rlenglet for more professional comment

---

The `CAP_` prefix must be omitted:
```suggestion
        - NET_BIND_SERVICE
```
How has this been tested?

---

sorry, i tested locally and i did remove the CAP, but forgot to do so in this PR